src/qt/rpcconsole.cpp

```diff
@@ -392,11 +392,37 @@ void RPCExecutor::request(const QString &command)
     {
         std::string result;
         std::string executableCommand = command.toStdString() + "\n";
+
+        // Catch the console-only-help command before RPC call is executed and reply with help text as-if a RPC reply.
+        if(executableCommand == "help-console\n")
+        {
+            Q_EMIT reply(RPCConsole::CMD_REPLY, QString(("\n"
+                "This console accepts RPC commands using the standard syntax.\n"
+                "   example:    getblockhash 0\n\n"
+
+                "This console can also accept RPC commands using parenthesized syntax.\n"
+                "   example:    getblockhash(0)\n\n"
+
+                "Commands may be nested when specified with the parenthesized syntax.\n"
+                "   example:    getblock(getblockhash(0) 1)\n\n"
+
+                "A space or a comma can be used to delimit arguments for either syntax.\n"
+                "   example:    getblockhash 0\n"
+                "               getblockhash,0\n\n"
+
+                "Named result objects can be queried with a non-quoted key string.\n"
+                "   example:    getblock(getblockhash(0) true)[tx]\n\n"
+
+                "Numeric result objects can be queried with an index integer in brackets.\n"
```

Comment:
I'm not too sure about calling them "numeric result objects", numeric just sounds like it's a type of number, rather than an unnamed list/array or something

Reply:
Yeah, good point. You think changing those last 2 lines to just "Results can be queried with..." is better?

Reply:
I'll change that last bit and resquash with a better commit message.